src/behaviors/loading/loading.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+'use strict';
+
+angular
+  .module('semantic.ui.behaviors.loading', [])
+
+  .directive('smLoading', smLoading);
+
+  smLoading.$inject = ['$parse'];
+
+  function smLoading($parse) {
+    return {
+      restrict:'A',
+      link: function(scope, element, attrs) {
+        scope.$watch(function() {
+          return $parse(attrs.smLoading)(scope);
+        }, function(loading) {
+          if (loading) {
+            element.addClass('loading');
```

Comment:
Another API you could consider is actually using a promise directly to enable/disable the loading class, which is something ngClass definitely cannot do, and is more in tune with the way people are writing web apps. An angular2 style version using rx observables would be cool too